fairroot/geobase/FairGeoCone.cxx

```diff
@@ -40,10 +40,11 @@
 #include <TArrayD.h>     // for TArrayD
 #include <TMathBase.h>   // for Abs
 #include <TString.h>     // for TString
+#include <fmt/core.h>    // for format
 #include <fstream>
 #include <ostream>    // for fstream, etc
 #include <stdio.h>    // for printf, sprintf, sscanf
-#include <string.h>   // for strlen
+#include <string.h>   // for string, strlen
```

Comment:
Note `<string.h>` and [`<string>`](https://en.cppreference.com/w/cpp/header/string) are different headers.

And I think, we should replace `<string.h>` by [`<cstring>`](https://en.cppreference.com/w/cpp/header/cstring)